### ISSUE TITLE
Feature Request:Org chart - multiple parents

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-api-issues#507]
Original [issue 511](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511) created by orwant on 2011-01-27T07:42:03.000Z:

<b>What would you like to see us add to this API?</b>
To be able to set more than one parent to the org unit.

<b>What component is this issue related to (PieChart, LineChart, DataTable,</b>
<b>Query, etc)?</b>

Org Chart

<b>*********************************************************</b>
<b>For developers viewing this issue: please click the 'star' icon to be</b>
<b>notified of future changes, and to let us know how many of you are</b>
<b>interested in seeing it resolved.</b>
<b>*********************************************************</b>

function drawVisualization() {
  // Create and populate the data table.
  var data = new google.visualization.DataTable();
  data.addColumn('string', 'Name');
  data.addColumn('string', 'Manager');
  data.addColumn('string', 'ToolTip');
  data.addRows(5);
  data.setCell(0, 0, 'Mike');
  data.setCell(0, 2, 'The President');
  data.setCell(1, 0,
      'Jim', 'Jim&lt;br/&gt;&lt;font color=&quot;red&quot;&gt;&lt;i&gt;Vice President&lt;i&gt;&lt;/font&gt;');
  data.setCell(1, 1, 'Mike');
  data.setCell(2, 0, 'Alice');
  data.setCell(2, 1, 'Mike');
  data.setCell(3, 0, 'Bob');
  data.setCell(3, 1, 'Jim,Alice');
  data.setCell(3, 2, 'Bob Sponge');
  data.setCell(4, 0, 'Carol');
  data.setCell(4, 1, 'Bob');

  // Create and draw the visualization.
  new google.visualization.OrgChart(document.getElementById('visualization')).
      draw(data, {allowHtml: true});
}

**Comment by orwant** [comment]
Comment [#1](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c1) originally posted by orwant on 2011-01-27T08:19:47.000Z:

Issue 509 has been merged into this issue.

**Comment by orwant** [comment]
Comment [#2](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c2) originally posted by orwant on 2011-01-27T18:16:12.000Z:

Hi. 
   I see that the same feature was requested by one more person. Is there any idea on when we should be able to see that feature ? Like couple of weeks or couple of months or more than that. Just an idea 

Regards
Meghashyam

**Comment by orwant** [comment]
Comment [#3](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c3) originally posted by orwant on 2011-04-28T15:47:15.000Z:

I really need to be able to display an org chart where child nodes can have multiple parents.  If this request will go through soon, it would really save my life.

**Comment by orwant** [comment]
Comment [#4](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c4) originally posted by orwant on 2011-07-24T00:53:45.000Z:

This is a duplicate request of issue 162.

**Comment by orwant** [comment]
Comment [#5](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c5) originally posted by orwant on 2011-07-24T07:20:19.000Z:

Right.

**Comment by orwant** [comment]
Comment [#6](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c6) originally posted by orwant on 2011-08-08T10:22:03.000Z:

Hello,
Has somebody found an alternative solution with org chart or another API to have mutiple parents ?
Regards,

**Comment by orwant** [comment]
Comment [#7](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c7) originally posted by orwant on 2012-02-23T13:03:48.000Z:

has sombody found alternative solution for org chart to have multiple parents

**Comment by orwant** [comment]
Comment [#8](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c8) originally posted by orwant on 2013-04-15T15:09:44.000Z:

I'm also looking for a solution to enable multiple parents with orgchart... so sad to find that nothing seems to have been made for this since 2011...

Would really be usefull...

**Comment by orwant** [comment]
Comment [#9](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c9) originally posted by orwant on 2014-01-02T17:56:57.000Z:

Would love to see this enhancement. Having more than one parent will help in lot of my implementations.

**Comment by orwant** [comment]
Comment [#10](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c10) originally posted by orwant on 2014-04-22T07:34:49.000Z:

Yes I am waiting on this. Really need this for this application I am building.
*

**Comment by orwant** [comment]
Comment [#11](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=511#c11) originally posted by orwant on 2014-12-28T19:17:15.000Z:

Hello guys, is there a solution for the multiple parents problem?
